src/borrow_check/region_inference.md

```diff
@@ -112,9 +112,7 @@ common sorts of constraints are:
    (e.g. `'a: 'b`). Outlives constraints are generated by the [MIR type
    checker].
 2. Liveness constraints. Each region needs to be live at points where it can be
-   used. These constraints are collected by [`generate_constraints`].
-
-[`generate_constraints`]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_borrowck/constraint_generation/fn.generate_constraints.html
+   used.
```

Comment:
Refactoed in https://github.com/rust-lang/rust/pull/118216 and the logic seems changed. I guess we have to update the sentence but this just removes the link for now.